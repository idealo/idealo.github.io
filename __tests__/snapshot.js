import React from "react";
import renderer from "react-test-renderer";
import Index from "../pages/index";
jest.mock(
  "next/image",
  () =>
    function Image({ src, alt }) {
      // eslint-disable-next-line @next/next/no-img-element
      return <img src={src} alt={alt} />;
    }
);
it("renders homepage unchanged", () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});
