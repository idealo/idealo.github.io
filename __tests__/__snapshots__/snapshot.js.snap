// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders landingpage unchanged 1`] = `
<div
  className="container"
>
  <main
    className="app__container"
  >
    <nav
      className="navigation__container"
    >
      <div
        className="navigation__list"
      >
        <span
          className="navigation__item   navigation__itemlogo"
        >
          <img
            alt="idealo.de Logo"
            className="navigation__logo"
            height="50px"
            src="/___mocks__/image-mock.png"
            width="100px"
          />
        </span>
        <a
          aria-label="Jobs at idealo"
          className="navigation__item"
          href="https://jobs.idealo.de"
          rel="noreferrer"
          target="_blank"
        >
          Jobs
        </a>
      </div>
    </nav>
  </main>
</div>
`;
